syntax = "proto3";

message AclRules {
  repeated AclRule rules = 1;
}

message AclRule {
  string src_mac = 1;
  string dst_mac = 2;
  repeated ActionRule action_rules = 3;
}

message ActionRule {
  string match_rule = 1;
  Action action = 2;

  enum Action {
    DENY = 0;
    ALLOW = 1;
  }
}